webapp/security/views.py

```diff
@@ -1,6 +1,6 @@
 # Standard library
 import re
```

Comment:
On line `284`  `notice.references.clear()` might not work anymore since it's not a m2m anymore.